src/js/common/schemaform/helpers.js

```diff
@@ -252,6 +252,11 @@ export function stringifyFormReplacer(key, value) {
     if (value.widget === 'autosuggest') {
       return value.id;
     }
+
+    // Exclude file data
+    if (value.confirmationCode && value.file) {
```

Comment:
Why are we using the confirmationCode to short circuit here?

Reply:
I'm using that to make sure we're in the typical file object and not something else that happens to have a `file` property.